todo_app/templates/index.html

```diff
@@ -12,7 +12,23 @@ <h1 class="display-4">To-Do App</h1>
       <h2>Items</h2>
       <ul class="list-group mb-4">
         <!-- List the todo items here -->
+        {% for ToDo in ToDos %}
+        <h1><a href="/todo/{{ToDo.id}}">{{ToDo.id}}</a> {{ToDo.title}}</h1>
+        <p>{{ToDo.status}}</p>
```

Comment:
For fully semantic HTML, each element in a list (the `ul` on line 13 stands for unordered list) should be a list item `<li>...</li>`
You can put whatever you want inside the `li` element.